apps/frontend/src/libs/components/input/styles.module.css

```diff
@@ -64,6 +64,7 @@
 
 .input-icon-left {
 	left: 10px;
+	display: flex;
```

Comment:
If this really help, then I think you should add this to `.input-icon-right` as well